### PR TITLE
Walk addition using regex

### PR DESCRIPTION
**Pull request by AdamMJenks**
This is the old walk addition, wanted to add it to the commit now but will be changed to use the link within the header

### CODE REVIEW

File: stattlepy/Stattleship_API.py [hunk 1]
```diff
@@ -42,22 +60,57 @@ def ss_get_results(self,**kwargs):
...
+                        ### Use try and except to see if the next link was found within the header
+                        try:
+                            if verbose:
+                                print 'Next link sent to API:'
+                                print next_link[0]
+
+                            headers = {
+                            'Authorization': token,
+                            'Accept':'application/vnd.stattleship.com; version=%s' %version,
+                            'Content-Type':'application/json'
```
Comment (dthyresson) [inline stattlepy/Stattleship_API.py]: @AdamMJenks We've started to measure analytics on each UserAgent so we know which libraries are using the API.

Could you:

1) Version this package (0.0.1)
2) Include a User-Agent header in the request:

```
        'User-Agent':
          "Stattleship Python/#{your version} (#{PLATFORM})"
```

As in

`Stattleship R/0.0.4 (x86_64-w64-mingw32/x64 (64-bit))`

Where `PLATFORM` is `(x86_64-w64-mingw32/x64 (64-bit))`

**Comment by dthyresson** [comment]
@AdamMJenks If you could add the user agent, should be good to merge. Thanks!

**Comment by dthyresson** [comment]
Also, noticed this:

```
setup(
      name='stattlepy',           name='stattlepy',
 -    version='0.1dev',      +    version='0.0.1',
      author = 'Adam Jenkins',            author = 'Adam Jenkins',
      packages=['stattlepy'],             packages=['stattlepy'],
      license='MIT',              license='MIT',
```

Could you include `Stattleship` as an additional author.

File: stattlepy/Stattleship_API.py (hunk 1 above)
Comment (AdamMJenks) [inline stattlepy/Stattleship_API.py]: For the User-Agent, does it matter where in the header it is positioned, i'm assuming at the end?

**Comment by dthyresson** [comment]
Thanks! Merging.

**Pull request merged**